CHANGELOG.md

```diff
@@ -11,7 +11,7 @@
 - Consume `OverdueSearchViewEffect` in `OverdueSearchEffectHandler`
 - Implement query to search overdue patients
 - Open overdue search screen when search button is clicked in overdue screen
-- Search for overdue patients when search query is valid
+- Show search results when overdue search results are loaded
```

Comment:
Are we updating the older CHANGELOG entry instead of creating a new one?

Reply:
That's a mistake